Makefile

```diff
@@ -89,6 +89,11 @@ cri-containerd:
 cc-containerd:
 	bash integration/containerd/confidential/run_tests.sh
 
+# Run the Confidential Containers tests for kubernetes.
+cc-kubernetes:
```

Comment:
We are almost there! :)

See in http://jenkins.katacontainers.io/job/wainer-tests-CCv0-ubuntu-20.04-x86_64-CC_CRI_CONTAINERD_K8S/3/console that the execution failed because bats was not installed in the environment:
```
08:15:26 integration/kubernetes/run_kubernetes_tests.sh: line 109: bats: command not found
08:15:26 [run_kubernetes_tests.sh:109] ERROR: bats k8s-attach-handlers.bats
```
You will need to install bats before run the tests. See the `kubernetes` Makefile target as an example.

Reply:
Awesome!!
We've added in the `install_bats.sh` line now 🤞 😄